components/match2/wikis/dota2/matchgroup_input_custom.lua

```diff
@@ -420,17 +419,38 @@ function matchFunctions.isFeatured(match)
 	end
 
 	if
-		(opponent1.type == Opponent.team
-		and Earnings.calculateForTeam({team = opponent1.name, year = year}) >= _MIN_EARNINGS_FOR_FEATURED)
-		or (opponent2.type == Opponent.team
-		and Earnings.calculateForTeam({team = opponent2.name, year = year}) >= _MIN_EARNINGS_FOR_FEATURED)
+		opponent1.type == Opponent.team and
+		matchFunctions.getEarnings(opponent1.name, year) >= _MIN_EARNINGS_FOR_FEATURED
+	then
+		return true
+	elseif
```

Comment:
```suggestion
	or
```
Should have the same effect & same performance (if the first statement return true, second statement won't be run)